src/components/donation/Steps/common/Summary.tsx

```diff
@@ -46,44 +50,55 @@ export default function Summary({
       {props.preSplitContent}
 
       <dl
-        className={`text-navy-l1 py-3 gap-y-2 grid grid-cols-[1fr_auto] items-center justify-between border-y border-gray-l4 ${splitClass}`}
+        className={`text-navy-l1 grid grid-cols-[1fr_auto] items-center justify-between border-y border-gray-l4 divide-y divide-gray-l4 ${splitClass}`}
       >
-        {(props.tip || locked > 0) && (
```

Comment:
flatten from :
```
(condition1 || condition2) && (
<>
 <node/>
 <node/>
</>
)
```
to individual 
`condition1 && <node/>`
`condition2 && <node/>`